tsl/test/sql/data_node.sql

```diff
@@ -459,37 +459,37 @@ SELECT * FROM _timescaledb_catalog.chunk_data_node;
 -- some ERROR cases
 \set ON_ERROR_STOP 0
 -- Will error due to under-replication
-SELECT * FROM block_new_chunks('data_node_2');
--- can't block/allow non-existing data node
-SELECT * FROM block_new_chunks('data_node_12345', 'disttable');
-SELECT * FROM allow_new_chunks('data_node_12345', 'disttable');
+SELECT * FROM cordon_data_node('data_node_2');
+-- can't cordon/uncordon non-existing data node
```

Comment:
It tests both cordon and uncordon, which is why it said both. Would it work to say "Can't cordon and uncordon...."

Reply:
I am wonder if it is possible to pass `cordon` or `uncordon` instead of `cordon/uncordon`. I cannot find the source, which takes care of the error message.